src/view/workflow/index.js

```diff
@@ -243,7 +176,36 @@ export default View.extend({
     })
 
     return elems
+  },
+  getPositions () {
+    let positions = {}
+    this.cy.nodes().forEach(node => {
+
+      positions[node.data('id')] = node.position()
+    })
+    return positions
+  },
+  setPositions (positions) {
+    Object.keys(positions).forEach(id => {
+      this.cy.nodes().filter(`[id = "${id}"]`)[0]?.position(positions[id])
+    })
+  },
+  setStartNode (start_task_id) {
```

Comment:
si start_task_id esta definida como this.start_task_id

```suggestion
  setStartNode () {
```
para que le llega por parámetro????